### ISSUE TITLE
SCP Issues list is public

### ISSUE THREAD

**Issue by jorgeguzmanecon** [infowetrust/startup-nation#32]
This issue list is public and can be seen by anyone as is our proprietary code.  Can we move to private?

**Comment by infowetrust** [comment]
we can make this repository private with these consequences:

![image](https://user-images.githubusercontent.com/20571941/57096410-0e435c80-6cca-11e9-9388-b4a74eb4f151.png)

**Comment by jorgeguzmanecon** [comment]
Yup.. no concerns on losing anything.  Bigger concern is on keeping open our IP to the public., let's make private then.

If it needs to be paid, I am not sure how to transfer me control but we can do that too.

**Comment by infowetrust** [comment]
**one last concern**

right now it is very convenient to share a test link, e.g. https://infowetrust.github.io/startup-nation/kentucky, to our team. I'm not sure if we will lose that ability if we go private.

**Comment by infowetrust** [comment]
@jorgeguzmanecon I recommend we go private immediately before the event
